pdks/Sky130/scripts/export.py

```diff
@@ -2,9 +2,11 @@
 #
 # Exports PDK data in Vlsir schema format (protobufs).
 
-# This requires that Vlsir schema bindings and VlsirTools both be importable by
-# Python. For development you can invoke with:
-#   PYTHONPATH=../VlsirTools/:../bindings/python/ ./proto2spice.py
+# To run this without installation (maybe during development), you need to
+# specify paths to VlsirTools, the Vlsir proto bindings, Hdl21 and the Sky130
+# PDK package (this one):
+#   PYTHONPATH=/path/Hdl21/pdks/Sky130:/path/Hdl21:/path/Vlsir/VlsirTools/:/path/Vlsir/bindings/python/ \
```

Comment:
Oh god no don't tell them to do that!

Reply:
This is of course tongue in cheek; I think you can just ditch this comment altogether.  
Anyone in "dev mode" needs to have some way to run Hdl21 stuff - e.g. `pytest` - and must have such a setup.

Reply:
Oh woops I merged it

Reply:
NP